amis/build_ami.sh

```diff
@@ -43,28 +44,28 @@ fi
 case $os in
 all)
   for x in $available_os; do
-    packer build -machine-readable -var-file=packer_variables.json packer_$x.json
+    packer build -machine-readable -var-file=${cwd}/packer_variables.json ${cwd}/packer_$x.json
     RC=$?
   done
   ;;
 centos6)
```

Comment:
since all the OSes are executing the same command we could compact the code
`centos6 | centos7 | alinux | ...`